package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@equinor/amplify-components",
-  "version": "7.12.9",
+  "version": "7.12.10",
```

Comment:
I bumped to 10 on my bugfix that was just merged. so i guess if you need a new build you need to bump again